src/pages/how-to/manage-network-access.mdx

```diff
@@ -1,99 +1,88 @@
 
 # Manage network access
-NetBird allows administrators to restrict access to resources (peers) by creating access rules and
-defining what peer groups are permitted to establish connections with one another. Rule can allow connections
-by specific protocol and ports.
+NetBird enables administrators to oversee and manage access between resources (peers) through access policies. These policies specify which peers and peer groups are permitted to connect to each other, detail the protocols and ports for these connections, and offer the option to include posture checks to apply zero trust principles, helping to adapt access control to specific contexts.
 
 ## Introduction
-A NetBird account comes with a `Default` rule that allows all peers of the account to connect to each other by all protocols,
-forming a full mesh network. In most cases, this is the desired state for a small network or network that has low-security requirements.
-When you need to restrict access to certain resources that belong to specific users or services within your organization,
-you can create rules that dictate who can access what.
+Initially, a NetBird account is configured with a `Default` policy which allows peers to connect via any protocol, resulting in the formation of a full mesh network. This setup often suits small networks or those requiring minimal security. In scenarios where higher security is needed or access to specific resources must be restricted for certain users or services, policies can be set up to determine access permissions.
```

Comment:
The main point in the frist sentence should reflect that machines connect to each other, or ALL to ALL and the second is protocol and ports. I had an impression that the emphasis is on protocol.